docs/using-gitbase/indexes.md

```diff
@@ -0,0 +1,16 @@
+# Indexes
+
+`gitbase` allows you to speed up queries creating indexes.
+
+Indexes are implemented as bitmaps using [pilosa](https://github.com/pilosa/pilosa) as a backend storage for them. To run a pilosa instance see the [getting started](./getting-started.md) section.
+
+Thus, to create indexes you must specify pilosa as the type of index.
+```
+CREATE INDEX files_path_idx ON files USING pilosa (file_path)
```

Comment:
I would add some example of using more than one column and using an expression.

Reply:
There are already examples for that in `examples.md` and a link to it. Should I add them here anyway?

Reply:
I added the specific section in `examples.md` to the link, btw

Reply:
I would say if we have an example here, complete it with all the possible index cases.

Reply:
If it is not complete, I would say to delete it to do not have duplicate info.

Reply:
So I'll remove it then and place here the link to the examples.